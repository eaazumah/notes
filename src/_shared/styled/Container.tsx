import styled from 'styled-components/native';

const Container = styled.View`
  flex: 1;
  background-color: ${props => props.theme.backgroundColor};
`;

export default Container;
