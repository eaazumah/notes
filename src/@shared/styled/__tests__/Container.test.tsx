import 'jest-styled-components';
import React from 'react';
import renderer from 'react-test-renderer';
import TestProviders from '../../../__testUtils__/Providers';
import Container from '../Container';

test('styled Container works', () => {
  const tree = renderer
    .create(
      <TestProviders mode={'light'}>
        <Container />
      </TestProviders>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
