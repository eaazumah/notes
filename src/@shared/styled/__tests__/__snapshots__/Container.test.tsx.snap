// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styled Container works 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#F5F5F5",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
/>
`;
